chatbot_app/api/routers/user_router.py

```diff
@@ -77,3 +79,17 @@ def login(login_request: LoginRequest, db: Session = Depends(get_db)) -> Token:
         data={"sub": auth_result.email, "role": auth_result.role}
     )
     return {"access_token": access_token, "token_type": "bearer", "status": "success"}
+
+
+@user_router.post("/register")
+def register(
```

Comment:
raczej tego typu endpointy sie robi bez body w odpowiedzi. Po prostu zwracasz 200. Bo frontend nie bedzie tego komunikatu nigdzie przetwarzal ani pokazywal nikomu. Takze olalbym to body zwracane i dal zwykle 200